Cabal-syntax/src/Distribution/Fields/ParseResult.hs

```diff
@@ -160,37 +154,3 @@ parseFatalFailure' = PR pr
     pr s                    failure _success = failure s
 
     err = PError zeroPos "Unknown fatal error"
-
--- | Helper combinator to do parsing plumbing for files.
---
--- Given a parser and a filename, return the parse of the file,
--- after checking if the file exists.
---
--- Argument order is chosen to encourage partial application.
-readAndParseFile
```

Comment:
Moving these seems wrong. (probably related that this module have to exposed?)

Reply:
So I moved these because "Verbosity" seems like a cabal-install specific thing as opposed to a Cabal-syntax thing, so I left it in `Cabal`.

These seem to be mostly for nice error reporting in a context where "Verbosity" makes sense, but I would guess most external use cases would rather handle the errors differently.

Reply:
Good point about `Verbosity`.